site/content/book/v2/mobileagents.md

```diff
@@ -0,0 +1,32 @@
+# Dealing with mobile agents
+
+Mobile agents are a key component in many decentralized identity solutions. Individuals holding Identity Wallets on their mobile phones, and IoT devices transmitting secure information from remote locations are some common examples of mobile agents. The ability to roam from different networks and change connectivity seamlessly is a great feature. However it comes at the expense that agents are unable to declare a unique network address where to be reached out. Also, that obstacle becomes more complex when agents are hidden inside network firewalls.
```

Comment:
"where to be reached out" -- > "where to be reached"

Reply:
Thanks. Fixed. And please correct/change all language matters and errors since English is not my native language.